platform/core/src/types/index.ts

```diff
@@ -12,12 +12,13 @@ export * from '../services/CustomizationService/types';
 // Separate out some generic types
 export * from './AppConfig';
 export * from './Consumer';
-export * from './Command';
+export type * from './Command';
 export * from './DisplaySet';
 export * from './StudyMetadata';
 export * from './PanelModule';
 export * from './IPubSub';
 export * from './Color';
+export type * from '../services/ToolbarService/ToolbarService';
```

Comment:
why not others?

Reply:
Those were the ones I was touching.  The lint used to complain about it, and now it wants them exported as types.

Reply:
at least put a todo that the rest should be done (it is easy just do it)